packs/equipment-effects/effect-desolation-locket-armor.json

```diff
@@ -4,10 +4,10 @@
     "name": "Effect: Desolation Locket - Armor",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Desolation Locket]</p>\n<p>You gain a +2 item bonus to saving throws against emotion effects.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Desolation Locket]</p>\n<p>You gain an item bonus to saving throws against emotion effects. (<em>major</em>) When you critically fail a saving throw against an emotion effect, you get a failure instead.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Desolation Locket]</p>\n<p>You gain an item bonus to saving throws against emotion effects.</p>\n<p>When you critically fail a saving throw against an emotion effect, you get a failure instead (<em>major</em>).</p>"
```

Reply:
We should standardize on this. How about this? I styled it like "Effect: Clockwork Goggles".